docs/blog/2019-06-26-live-preview-for-drupal/index.md

```diff
@@ -0,0 +1,82 @@
+
+Two types of people will be excited about this article.
+
+1. The developer who has been eyeing Gatsby but their clients are hesitant to leave an established workflow that includes previewing their work.
+
+2. A content focused client who wants their site to be cutting edge, but doesn't want to leave their comfortable workflow that allows them to rapidly toggle between content changes and the visual impact of those changes on the website.
+
+As Sam Bhagwat wrote [when announcing Gatsby Preview](blog/2018-07-17-announcing-gatsby-preview/):
+
+> Content teams are creators. And creators, as Bret Victor put it, “need an immediate connection to what they’re making.”
+
+> When a copywriter edits a headline in their CMS, they shouldn’t have to imagine what their change looks like in context. They should see it—immediately.
+
+At [Third and Grove](https://www.thirdandgrove.com/), we've been working on a preview module for Drupal to be used with Gatsby Cloud.
+
+**Today, we're excited to announce that we're beta launching live preview with Drupal and Gatsby!**
+
+With modern JavaScript, React’s component model, and Progressive Web App features built in, Gatsby + Gatsby Preview is a compelling alternative to traditional CMS site building, while still allowing content teams to use mature CMS content workflows. So far, it's been available for Contentful, Sanity, DatoCMS, and Contentstack.
+
+Today's launch of Gatsby Preview + Drupal is a major milestone, making Gatsby Preview available for a huge, thriving open-source community.
+
+Here's how it works:
+
+## Getting started
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/H72PY3wNMcI" frameBorder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowFullScreen title="Youtube video: Drupal Gatsby Preview setup walkthrough"></iframe>
+
+### Step 1: Sign Up for Gatsby Preview Beta
+
+You will need to sign up for a Gatsby cloud preview beta account, so this is a meta beta. You can sign up for a [Gatsby cloud preview 14 day trial here](https://www.gatsbyjs.com/preview/?_ga=2.156650491.1704520703.1561474285-32798346.1550767689) and get more information on the [Gatsby blog](/blog/2019-03-22-introducing-gatsby-preview-beta/).
+
+### Step 2: Turn on the Preview Flag in Gatsby Preview
+
+The gatsby source plugin is available on [npm under the package name `gatsby-source-drupal-preview`](https://www.npmjs.com/package/gatsby-source-drupal-preview). Setup is exactly the same as the Drupal source plugin, however, you now have a `preview` flag to turn on in the options object:
+
+```javascript:title=gatsby-config.js
+module.exports = {
+  plugins: [
+    {
+      resolve: `gatsby-source-drupal-preview`,
+      options: {
+        baseUrl: `...`,
+        preview: true,
+      },
+    },
+  ],
+}
+```
```

Comment:
Note on this - Grant's code was merged into main `gatsby-source-drupal` plugin. It also doesn't require setting `preview: true` anymore - it's just enabled by default.

But it's not yet completely finished - please see notes in https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-source-drupal/README.md#gatsby-preview-experimental - there are features missing, and we will likely iterate on APIs (possibly making breaking changes in the process)